src/server/json_family.cc

```diff
@@ -778,6 +778,39 @@ OpResult<vector<OptSizeT>> OpArrInsert(const OpArgs& op_args, string_view key, s
   return vec;
 }
 
+// Returns numeric vector that represents the new length of the array at each path, or Null reply 
+// if the matching JSON value is not an array.
```

Comment:
This line is redundant. I'd write only the return value type after a successful operation.